rfc-144-consistent-cache-expiry.md

```diff
@@ -0,0 +1,186 @@
+# 144 More reliable emergency banner deployment
+
+Author: Bill Franklin\
+Date: November 2021\
+Deadline: 26 November 2021
+
+## Summary
+
+When we publish the [emergency banner], we clear several caches which contain
+the Static templates, so that the banner will appear on www.gov.uk immediately.
```

Comment:
I suggest this is changed to
>with the intention that the banner will appear across www.gov.uk immediately

because a)we know it currently doesn't happen immediately and b)this could be taken to mean the homepage only

Reply:
Thanks, I've made this change.